### ISSUE TITLE
Allow linters to be disabled/enabled with inline code comments

### ISSUE THREAD

**Issue by emiloberg** [brigade/scss-lint#150]
Do I have any way of controlling the linting inside an scss-file, such as the jslint `/*ignore jslint start*/`

Example:
This mixin has a `*zoom` property which is a hack and not valid (s)css. Can I tell scss-lint "I know you're upset about this, but just at this instance, ignore this"?

``` SCSS
@mixin clearfix {
  //ignore scsslint start
  *zoom: 1;
  //ignore scsslint end

  &:before,
  &:after {
    display: table;
    content: '';
    line-height: 0;
  }

  &:after {
    clear: both;
  }
}
```

(Yes I know that in this specific case, I can add `*zoom` to the extra_properties of the PropertySpelling lint rule)
